src/main/java/org/broadinstitute/consent/http/db/DarCollectionSummaryDAO.java

```diff
@@ -172,60 +232,6 @@ WHERE LOWER(election.election_type) = 'dataaccess'
       """)
   List<DarCollectionSummary> getDarCollectionSummariesForAdmin();
 
-  @RegisterBeanMapper(value = DarCollectionSummary.class)
-  @RegisterBeanMapper(value = DarCollection.class)
-  @RegisterBeanMapper(value = Election.class)
-  @UseRowReducer(DarCollectionSummaryReducer.class)
-  @SqlQuery("""
-    SELECT
-        c.collection_id AS dar_collection_id,
-        c.dar_code,
-        latest_dar.submission_date,
-        latest_dar.reference_id AS latest_dar_reference_id,
-        latest_dar.parent_id AS latest_dar_parent_id,
-        u.display_name AS researcher_name,
-        i.institution_name,
-        e.election_id,
-        e.status,
-        e.dataset_id,
-        e.reference_id AS election_reference_id,
-        dd.dataset_id AS dd_datasetid,
-        (regexp_replace(latest_dar.data #>> '{}', '\\u0000', '', 'g'))::jsonb ->> 'projectTitle' AS name,
-        (regexp_replace(latest_dar.data #>> '{}', '\\u0000', '', 'g'))::jsonb ->> 'status' AS dar_status,
```

Comment:
It looks like some `\\\\` were replaced with `\\`. The 4 backslashes were added in https://github.com/DataBiosphere/consent/pull/2156 so this PR fixes these unintentional changes.